app/src/main/java/com/example/airbnb/presentation/explore/ExploreFragment.kt

```diff
@@ -1,11 +1,81 @@
 package com.example.airbnb.presentation.explore
 
+import android.content.Intent
+import androidx.core.content.ContextCompat
 import com.example.airbnb.R
 import com.example.airbnb.core.base.BindingFragment
+import com.example.airbnb.data.DummyExploreImageList
 import com.example.airbnb.databinding.FragmentExploreBinding
+import com.example.airbnb.presentation.where.WhereActivity
+import com.google.android.material.tabs.TabLayoutMediator
 
 class ExploreFragment : BindingFragment<FragmentExploreBinding>(R.layout.fragment_explore) {
+
+    private lateinit var exploreViewPagerAdapter: ExploreViewPagerAdapter
+
     override fun initView() {
-        //로직 작성
+        // 로직 작성
+        val dummyExploreImageList = DummyExploreImageList.dummyExploreImage
+        exploreViewPagerAdapter = ExploreViewPagerAdapter(dummyExploreImageList)
+
+        binding.run {
+            // 뷰페이저 어댑터 연결
+            vpExplore.adapter = exploreViewPagerAdapter
+            indicatorViewpagerImageDots.attachTo(binding.vpExplore)
+
+            // 뷰페이저 화면 디자인
+            vpExplore.offscreenPageLimit = 3
+            vpExplore.setPadding(40, 0, 40, 0)
+
+            val tabTitles = listOf("통나무집", "방", "최고의 전망", "해변 근처")
+            val tabImage = listOf(
+                R.drawable.ic_explore_treehouse,
+                R.drawable.ic_explore_room,
+                R.drawable.ic_explore_goodview,
+                R.drawable.ic_explore_seashore,
+            )
+
+            // tab, 뷰페이저 연결
+            TabLayoutMediator(tapNavMain, vpExplore) { tab, position ->
+                when (position) {
+                    0 -> {
+                        tab.text = tabTitles[position]
+                        tab.icon = ContextCompat.getDrawable(
+                            requireContext(),
+                            tabImage[position],
+                        )
+                    }
+
+                    1 -> {
+                        tab.text = tabTitles[position]
+                        tab.icon =
+                            ContextCompat.getDrawable(requireContext(), tabImage[position])
+                    }
+
+                    2 -> {
+                        tab.text = tabTitles[position]
+                        tab.icon = ContextCompat.getDrawable(
+                            requireContext(),
+                            tabImage[position],
+                        )
+                    }
+
+                    3 -> {
+                        tab.text = tabTitles[position]
+                        tab.icon = ContextCompat.getDrawable(
+                            requireContext(),
+                            tabImage[position],
+                        )
+                    }
+                }
+            }.attach()
```

Comment:
```suggestion
            TabLayoutMediator(tapNavMain, vpExplore) { tab, position ->
               tab.text = tabTitles[position]
               tab.icon = ContextCompat.getDrawable(requireContext(), tabImage[position])
            }.attach()
```

tab, position마다 로직이 똑같으니깐, 요렇게 작성해도 될듯합니다!

Reply:
오우 굿굿

Reply:
반영했습니다~! 확실히 깔끔해진다,,,,,,